static/js/appbundle.js

```diff
@@ -103478,7 +103478,7 @@ def note_with_error(value, err):
         contentType: "application/json",
         dataType: "json"
       }).done(function(response) {
-        if (response.achievement) {
+        if (response && response.achievement) {
```

Comment:
This check is necessary because deleting a class returns `make_response()` and this would give errors because then response doesn't exist. I'm not sure why.